work_queue/src/tasks.txt

```diff
@@ -0,0 +1,5 @@
+{ "command_line" : "sed s/number/No./g file1.txt > file1.replaced" , "output_files" : [ { "local_name" : "file1.replaced" , "remote_name" : "file1.replaced" , "flags" : { "WORK_QUEUE_CACHE": true , "WORK_QUEUE_NOCACHE" : false , "WORK_QUEUE_WATCH" : false } } ] , "input_files" : [ { "remote_name": "sed" , "local_name" : "/usr/bin/sed" , "flags" : { "WORK_QUEUE_CACHE" : false , "WORK_QUEUE_NOCACHE" : true , "WORK_QUEUE_WATCH" : false } } , { "remote_name" : "file1.txt" , "local_name" : "file1.txt" , "flags" : { "WORK_QUEUE_CACHE" : false , "WORK_QUEUE_NOCACHE" : true , "WORK_QUEUE_WATCH" : false } } ] }
```

Comment:
In the C interface, all of the symbolic constants had to have `WORK_QUEUE` appended to the front in order to ensure there would not be namespace conflicts.  That's a bit repetitive here, so let's have the JSON API just look for simple names like `"cache" : true` etc.

Also note that `WORK_QUEUE_CACHE` and `WORK_QUEUE_NOCACHE` are antonyms, so really only the first one is needed.